hub/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .api.dataset import Dataset
+from .api.load import load
```

Comment:
are we sure we want to keep this load? can be confusing and seem like we're loading a dataset. I think read is better.

Reply:
can rename later, should talk about it in an api review